examples/FieldLevelValidation.js

```diff
@@ -7,7 +7,7 @@ const FieldLevelValidation = () => (
   <div>
     <h1>Pick a username</h1>
     <Formik
-      initialValues={{ username: '', email: '' }}
+      initialValues={{}}
```

Comment:
You still should do this unless we give Field's value a default of `''`.

Reply:
The thing is, if you use the props-only API, `initialValues` prop really isn't optional.

Reply:
Agree.